packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -51,6 +51,16 @@ constexpr auto log_inserted_key = std::array<char, 1>{-60};
 constexpr auto send_inserted_key = std::array<char, 1>{-62};
 constexpr auto schema_version_key = std::array<char, 1>{-64};
 constexpr auto logscursor_current_prefix = std::array<char, 1>{-120};
+
+template <class T>
+void printResult(const char* msg, ValueResult<T> result) {
```

Comment:
`const ValueResult<T>& result` to save the copy

Reply:
:+1: